scripts/download-pokemon-data.js

```
@@ -0,0 +1,183 @@
+/**
+ * Download Pokemon TCG data from GitHub
+ * This script downloads all sets and cards data from the Pokemon TCG GitHub repository
+ * and saves them to the public/data directory for local access.
+ */
+
+const fs = require('fs');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)